README.md

```diff
@@ -27,130 +11,77 @@ single interface.
 It was originally developed at OpenHub, and is used to generate
 the reports at www.openhub.net.
 
-## System Requirements
-
-One could use the bundled Dockerfile to test Ohloh SCM in a container and skip
-this section entirely. See [docker](https://github.com/blackducksoftware/ohloh_scm/#using-docker).
-
-Ohloh SCM is developed on Mac OS X 10.13.6(High Sierra) and Ubuntu 18.04 LTS.
-Other Linux environments should also work, but your mileage may vary.
-
-Ohloh SCM does not support Windows.
-
-Ohloh SCM targets Ruby 2.3 and Rake 12.3.
-
-Ohloh SCM interfaces with CVSNT, Subversion, Git and Mercurial through the
-shell. In order to pass the unit tests, all three systems must be installed
-and on your path. Ohloh is currently tested on the following versions:
-
-cvsnt 2.5.03
-svn 1.9.7
-git 2.17.1
-hg 4.5.3
-bzr 2.8.0
-
-If you are using CVS instead of CVSNT, you can potentially try creating
-a shell alias or symlink mapping 'cvsnt' to 'cvs'.
-
-Ohloh SCM uses [posix-spawn](https://github.com/rtomayko/posix-spawn) to
-execute commands so ensure *posix-spawn* gem is installed
```

Comment:
We can get rid of the posix-spawn info, presuming we have done away wit that, but keep as much of the remaining information as possible,

Reply:
We dont use posix-spawn anymore. So we do not need that information here.
The symlink to cvsnt has been covered in the *Development Setup* instructions.